docs/en/observability/view-observability-alerts.asciidoc

```diff
@@ -89,8 +89,8 @@ You can also use the toolbar buttons in the upper-right to customize the display
 [[cases-observability-alerts]]
 == Add alerts to cases
 
-From the Alerts table, you can add one or more alerts to a case. Select image:images/action-dropdown.png[Three dots used to expand the "More actions" menu,height=22]
-to add the alert to a new case or add it to an existing case. You can add an unlimited amount of alerts from any rule type.
+From the Alerts table, you can add one or more alerts to a case. Click the image:images/icons/boxesHorizontal.svg[More actions] icon
+to add the alert to a new or existing case. You can add an unlimited amount of alerts from any rule type.
 
 NOTE: Each case can have a maximum of 1,000 alerts.
```

Comment:
Is this note contradictory to the "You can add an unlimited amount of alerts..." line above, or am I misreading?

Reply:
Good catch.  Looks like the previous sentence was overlooked when the note was added in https://github.com/elastic/observability-docs/pull/3738.